test/CheckoutSdkTest/CardIssuing/CardHolders/CardHoldersIntegrationTest.cs

```diff
@@ -0,0 +1,170 @@
+            response.Body.ShouldNotBeNull();
+            response.ResponseHeaders.ShouldNotBeNull();
+            response.Links.ShouldNotBeNull();
+            response.Cards.ShouldNotBeNull();
+            foreach (CardCardHolderResponse card in response.Cards)
+            {
+                card.CardholderId.ShouldBe(cardHolder.Id);
+                card.ClientId.ShouldBe("cli_p6jeowdtuxku3azxgt2qa7kq7a");
+            }
+        }
+
+        [Fact]
+        private async Task ShouldThrowErrorNotFoundCardHolderCards()
+        {
+            await AssertNotFound(_api.CardHoldersClient().GetCardHolderCards("not_found"));
+        }
+
+        private static CardHolderRequest CardHolderRequest()
+        {
+            CardHolderRequest request = new CardHolderRequest
+            {
+                Type = CardHolderType.Individual,
+                Reference = "X-123456-N11",
+                EntityId = "ent_mujh2nia2ypezmw5fo2fofk7ka",
+                FirstName = "John",
+                MiddleName = "Fitzgerald",
+                LastName = "Kennedy",
+                Email = "john.kennedy@myemaildomain.com",
+                PhoneNumber = GetPhone(),
+                DateOfBirth = "1985-05-15",
+                BillingAddress = GetAddress(),
+                ResidencyAddress = GetAddress(),
+                Document = new CardHolderDocument
+                {
+                    Type = DocumentType.Passport,
+                    FrontDocumentId = "file_6lbss42ezvoufcb2beo76rvwly",
+                    BackDocumentId = "file_aaz5pemp6326zbuvevp6qroqu4"
+                }
+            };
+            return request;
+        }
+
+        private static CardHolderRequest CardHolderBadRequest()
+        {
+            CardHolderRequest request = new CardHolderRequest
+            {
+                Type = CardHolderType.Individual,
+                Reference = "X-123456-N11",
+                EntityId = "ent_fa6psq242dcd6fdn5gifcq1491",
+                FirstName = "John",
+                MiddleName = "Fitzgerald",
+                LastName = "Kennedy",
+                Email = "john.kennedy@myemaildomain.com",
+                PhoneNumber = GetPhone(),
+                DateOfBirth = "1985-05-15",
+                BillingAddress = GetAddress(),
+                ResidencyAddress = GetAddress(),
+                Document = new CardHolderDocument
+                {
+                    Type = DocumentType.Passport,
+                    FrontDocumentId = "file_6lbss42ezvoufcb2beo76rvwly",
+                    BackDocumentId = "file_aaz5pemp6326zbuvevp6qroqu4"
+                }
+            };
+            return request;
+        }
+
+        private static CheckoutApi GetIssuingCheckoutApi()
+        {
+            return CheckoutSdk.Builder()
+                .OAuth()
+                .ClientCredentials(
+                    System.Environment.GetEnvironmentVariable("CHECKOUT_DEFAULT_OAUTH_ISSUING_CLIENT_ID"),
```

Comment:
You need to add this credentials in the github workflows too

Reply:
I don't have privileges, @armando-ibarra-cko can you add this credentials?